stock_picking_invoice_product_group/models/account_invoice.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Alex Comba - Agile Business Group
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import models, fields
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    related_picking_ids = fields.Many2many(
```

Comment:
@pedrobaeza, do you have any reference/example about migration scripts and build on them?
Are they also able to handle the case where the ```stock_picking_invoice_link``` is already installed and it simply needs an upgrade?

Any help would be appreciated.